hack/e2e.sh

```diff
@@ -44,6 +44,9 @@ trap cleanup EXIT
 export E2E_KUTTL_OSCLOUDS="${creds_dir}/clouds.yaml"
 export E2E_KUTTL_CACERT_OPT
 
+# default flavor name to use for tests
+E2E_KUTTL_FLAVOR=${E2E_KUTTL_FLAVOR:-m1.tiny}
```

Comment:
I think you need to export E2E_KUTTL_FLAVOR here, otherwise it won't be passed to kuttle when it runs.